Tests/scripts/hook_validations/docker.py

```diff
@@ -225,9 +236,8 @@ def parse_docker_image(docker_image):
         """
         if docker_image:
             try:
-                name = re.findall(r'(demisto\/.+):.+', docker_image, re.IGNORECASE)[0]
-                tag = re.findall(r'demisto\/.+:(.+)', docker_image, re.IGNORECASE)[0]
-                return name, tag
+                regex = re.findall(r'(demisto\/.+):(.+)', docker_image, re.IGNORECASE)
```

Comment:
`re.findall` returns list of tuples
```suggestion
                regex = re.findall(r'(demisto\/.+):(.+)', docker_image, re.IGNORECASE)[0]
```

Reply:
fixed even though test doesn't fails :D